halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Node.java

```diff
@@ -36,21 +32,6 @@
   @JsonIgnore
   public abstract NodeIterator getChildren();
 
-  @JsonIgnore
-  private Set<ComponentName> spinnakerComponents = new HashSet<>();
```

Comment:
This is where the halconfig model depended on the config generation by labeling itself with the which compenent's config to write.